components/dashboard/src/teams/git-integrations/GitIntegrations.tsx

```diff
@@ -6,11 +6,22 @@
 
 import { FunctionComponent } from "react";
 import { SpinnerLoader } from "../../components/Loader";
+import { Heading2, Subheading } from "../../components/typography/headings";
 import { useOrgAuthProvidersQuery } from "../../data/auth-providers/org-auth-providers-query";
 import { GitIntegrationsList } from "./GitIntegrationsList";
 
 export const GitIntegrations: FunctionComponent = () => {
     const { data, isLoading } = useOrgAuthProvidersQuery();
 
-    return <div>{isLoading ? <SpinnerLoader /> : <GitIntegrationsList providers={data || []} />}</div>;
+    if (isLoading) {
+        return <SpinnerLoader />;
+    }
+
+    return (
+        <div>
+            <Heading2>Git Auth configurations</Heading2>
```

Comment:
Adding a heading/subheading to the content of the git auth page so it's a bit more consistent w/ the SSO page layout.